rails-commenting.rb

```diff
@@ -5,27 +5,27 @@
 
 # app/controller/blog_posts_controller.rb
 
-# 1)
+# 1)This indicates the controller that we are in, in this case it controls the blog posts for the app
 class BlogPostsController < ApplicationController
   def index
-    # 2)
+    # 2) it assigns our @posts instance variable to be all the existing blog posts.
     @posts = BlogPost.all
   end
 
   def show
-    # 3)
+    # 3) assigns our instance variable to find a specific blog post by using it's given id.
     @post = BlogPost.find(params[:id])
   end
 
-  # 4)
+  # 4) it is a method that allows us to call our new blog post
```

Comment:
Kind of, again, this is something that would help create the connection between say a form and our post ability.  I would also want to add in more information as to why its empty, and why that is okay.